sys/src/cmd/4c/mkfile

```diff
@@ -29,9 +29,7 @@ enam.c:	4.out.h
 	rc mkenam
 
 $LIB:
-	rfork
```

Comment:
this worries me. 

which one is 4 again?

Reply:
4 is mips64.  4c has a custom version of lex.c  this part of the makefile differs from other ?c mkfiles because of this.  this change makes the 4c mkfile consistent with other ?c mkfiles.

the only change made in 4c/lex.c that hasn't been made in cc/lex.c is related to some already-mips-specific code.  since there's already mips-specific code in cc/lex.c, I've made the change in this PR.

we can make this change because in this PR we take the